x/metadata/wasm/encode.go

```diff
@@ -59,7 +59,8 @@ func (params *WriteScope) Encode() ([]sdk.Msg, error) {
 		return nil, err
 	}
 
-	msg := types.NewMsgWriteScopeRequest(*scope, params.Signers)
+	// TODO: should the structures be updated with expiration settings?
+	msg := types.NewMsgWriteScopeRequest(*scope, params.Signers, "1y")
```

Comment:
Unaddressed TODO.
Also, duplication of the default expiration string.

Reply:
added a new field `Expiration string` to `WriteScope` and passing that in. This will eventually default to `expiration types.DefaultDuration` based on https://github.com/provenance-io/provenance/pull/1115#discussion_r987949444